public/app/features/alerting/unified/components/rules/EditRuleGroupModal.test.tsx

```diff
@@ -49,7 +61,8 @@ describe('EditGroupModal', () => {
 });
 
 describe('EditGroupModal component on cloud alert rules', () => {
-  const promDsSettings = mockDataSource({ name: 'Prometheus-1', uid: 'Prometheus-1' });
+  const { dataSource, rulerConfig } = mimirDataSource();
+  setupDataSources(dataSource);
```

Comment:
Nit, but it could cause confusion: It looks like `mimirDataSource` calls `setupDataSources` itself. So we don't technically need to call this here, but perhaps it makes it clearer anyway